mocks/fixtures/decos-vergunningen-zaken-response.json

```diff
@@ -718,6 +718,67 @@
         "text9": "69BTR4"
       },
       "links": []
+    },
+    {
+      "key": "FEA9F714ED244867A0EE7386B7B11886",
+      "fields": {
+        "title": "<Een Status>",
+        "text45": "Varen ligplaatsvergunning",
+        "mark": "Z/24/2243238",
+        "dfunction": "<Afgehandeld | Geannuleerd>",
```

Comment:
Mockdata moet wel échte data representeren. In het "echt" krijgen we hier niet deze waarde. Je kunt er een met afgehandeld maken en een met geannuleerd?